tests/conftest.py

```diff
@@ -70,3 +73,361 @@ def _sample_all_wsis(_sample_ndpi, _sample_svs, _sample_jp2, tmpdir_factory):
+
+    return src_image_file_path
+
+
+@pytest.fixture(scope="session")
+def _norm_macenko(tmpdir_factory):
+    """Sample pytest fixture for norm_macenko image.
+    Download norm_macenko image for pytest."""
+
+    norm_macenko_file_path = tmpdir_factory.mktemp("data")\
+        .join("norm_macenko.png")
+
+    if not pathlib.Path(norm_macenko_file_path).is_file():
+        print("\nDownloading Norm_macenko image")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "stainnorm/norm_macenko.png")
+        with open(norm_macenko_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping Norm_macenko image")
+
+    return norm_macenko_file_path
+
+
+@pytest.fixture(scope="session")
+def _norm_reinhard(tmpdir_factory):
+    """Sample pytest fixture for norm_reinhard image.
+    Download norm_reinhard image for pytest."""
+
+    norm_reinhard_file_path = tmpdir_factory.mktemp("data")\
+        .join("norm_reinhard.png")
+
+    if not pathlib.Path(norm_reinhard_file_path).is_file():
+        print("\nDownloading norm_reinhard image")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "stainnorm/norm_reinhard.png")
+        with open(norm_reinhard_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping norm_reinhard image")
+
+    return norm_reinhard_file_path
+
+
+@pytest.fixture(scope="session")
+def _norm_ruifrok(tmpdir_factory):
+    """Sample pytest fixture for norm_ruifrok image.
+    Download norm_ruifrok image for pytest."""
+
+    norm_ruifrok_file_path = tmpdir_factory.mktemp("data")\
+        .join("norm_ruifrok.png")
+
+    if not pathlib.Path(norm_ruifrok_file_path).is_file():
+        print("\nDownloading norm_ruifrok image")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "stainnorm/norm_ruifrok.png")
+        with open(norm_ruifrok_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping norm_ruifrok image")
+
+    return norm_ruifrok_file_path
+
+
+@pytest.fixture(scope="session")
+def _norm_vahadane(tmpdir_factory):
+    """Sample pytest fixture for norm_vahadane image.
+    Download norm_vahadane image for pytest."""
+
+    norm_vahadane_file_path = tmpdir_factory.mktemp("data")\
+        .join("norm_vahadane.png")
+
+    if not pathlib.Path(norm_vahadane_file_path).is_file():
+        print("\nDownloading norm_vahadane image")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "stainnorm/norm_vahadane.png")
+        with open(norm_vahadane_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping norm_vahadane image")
+
+    return norm_vahadane_file_path
+
+
+@pytest.fixture(scope="session")
+def _tcga_he_7130(tmpdir_factory):
+    """Sample pytest fixture for TCGA-HE-7130-01Z-00-DX1 image.
```

Comment:
"""Sample pytest fixture for TCGA-HE-7130-01Z-00-DX1 image.
    Download TCGA-HE-7130-01Z-00-DX1 image for pytest.

"""